stock_move_location/wizard/stock_move_location.py

```diff
@@ -75,6 +63,32 @@ def _compute_readonly_locations(self):
                 rec.origin_location_disable = True
                 rec.destination_location_disable = True
 
+    @api.depends("origin_location_id")
+    def _compute_picking_type_id(self):
+        company_id = self.env.context.get("company_id") or self.env.user.company_id.id
```

Comment:
@AungKoKoLin1997 If you depends on company context key, add @api.depends_context("comapny")